lib/protocol/MiotDevice.js

```diff
@@ -1327,13 +1328,28 @@ class MiotDevice extends EventEmitter {
   // updates the property value with the value retrieved from the device
   _updatePropertyValueFromDevice(result, propName, response) {
     if (this._isResponseValid(response)) {
+      if (LUMI_MOTION_MODELS.includes(this.getModel())) {
```

Comment:
Why special handling for the device needs to be done?

Reply:
it is needed because this device will always return "value":true regardless of the motion detect status

if motion detection: false
curl: miot cloud get-props '[{"did":"lumi.XXXX","siid":2,"piid":1}]'
return: [{"did":"lumi.XXXX","iid":"0.2.1","siid":2,"piid":1,"value":true,"code":0,"updateTime":1691225681,"exe_time":0}]

if motion detection: true
curl: miot cloud get-props '[{"did":"lumi.XXXX","siid":2,"piid":1}]'
return: [{"did":"lumi.XXXX","iid":"0.2.1","siid":2,"piid":1,"value":true,"code":0,"updateTime":1691226209,"exe_time":0}]

the only value that changes is the "updateTime", but it also has some special rules about when exactly the value updateTime will be updated, so I am still working on it.

Reply:
You should in that case register for specific property changes and handle that in the device class itself. It is a little bit odd to do it in a central generic place...

Reply:
Sounds good, I might need to change the MiotProperty then to catch the "updateTime" tho, as I don't see it pass down to specific device handling. Or do you have a suggestion of any other ways to achieve it?

Reply:
Oh, i know what you mean now. "updateTime" is only there is the response, after processing that information is lost. I am thinking if maybe to offer an event would be a better solution,. Interested devices could then register for that event and do extra data processing.

Reply:
👍